internal/tiltfile/tiltfile_test.go

```diff
@@ -932,6 +932,20 @@ if True:
 	f.assertConfigFiles("Tiltfile", "foo/Dockerfile", "foo.yaml")
 }
 
+func TestTopLevelForLoop(t *testing.T) {
+	f := newFixture(t)
+	defer f.TearDown()
+
+	f.setupFoo()
+
+	f.file("Tiltfile", `
+for i in range(1, 3):
+	print(i)
+`)
+
+	f.load()
```

Comment:
this... doesn't test anything? are you just trying to make sure it doesn't crash?

Reply:
Yep!

Reply:
...okay 🤷‍♀️ can you leave a comment or rename the test to `TestTopLevelForLoopDoesntCrash` or something?

Reply:
@maiamcc for context this isn't allowed normally in Starlark, and is behavior I [specifically enabled](https://github.com/windmilleng/tilt/blob/master/internal/tiltfile/tiltfile.go#L27) to allow top level if statements and I wanted to make sure that it also applied to the for loop control structure.

Reply:
Oooh i see! (Worth noting that in a comment)